pkg/api/object.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2016 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+package api
+
+import (
+	"net/http"
+
+	"github.com/rook/rook/pkg/cephmgr/rgw"
+)
+
+// Creates a new object store in this cluster.
+// POST
+// /object
+func (h *Handler) CreateObjectStore(w http.ResponseWriter, r *http.Request) {
```

Comment:
is there any concept of a GET that returns information about the created object store?  or is that purpose better served by the API that I will add that gives you connection information?